package.json

```diff
@@ -9,7 +9,7 @@
     "npm": "8.5.5"
   },
   "devDependencies": {
-    "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git",
+    "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git#task/tup-151-news--center-article-layout",
```

Comment:
**Important**

After merge of https://github.com/TACC/Core-Styles/pull/21:
1.  Checkout branch.
2. Run `npm install --save-dev git+https://git@github.com/TACC/Core-Styles.git`.
3. Commit changes.

Reply:
Nope. Instead, I pinned to v0.5.1 (e6f3a06) because I am on top of Core-Styles versions now.[^1]

[^1]: If I get _off of the top of_ them, I should use commits _not_ branches.

Reply:
Now, NPM pacakge is used, so we point to a version.